src/components/detailPanel/DetailPanel.js

```diff
@@ -50,23 +58,32 @@ const DetailPanel = ({ isOpen, togglePanel, feature }) => {
       documents,
     } = feature.properties;
     const [lng, lat] = feature.geometry.coordinates;
+
     return (
       <div
         className={classNames(
           styles.Container,
           isOpen ? styles.ContainerOpen : ''
         )}
```

Comment:
ik vind die hele classNames werkwijze heel onhandig. wat vind jij?

Reply:
not my choice, normally I have no problem with it, but I try to use only styled components